lib/SPIRV/libSPIRV/SPIRVUtil.h

```diff
@@ -86,15 +86,15 @@ template <class Ty1, class Ty2, class Identifier = void> struct SPIRVMap {
   void init();
 
   static Ty2 map(Ty1 Key) {
-    Ty2 Val;
+    Ty2 Val = {};
```

Comment:
It is not entirely clear to me what was the problem here, but ok, I probably just don't know something about C++

Reply:
The problem is uninitialized `Val`, which is treated as error after importing HandleLLVMOptions